app/src/main/java/org/fossasia/openevent/general/event/EventService.kt

```diff
@@ -76,6 +94,8 @@ class EventService(
         return eventApi.searchEvents("name", locationName).flatMap { apiList ->
             val eventIds = apiList.map { it.id }.toList()
             eventTopicsDao.insertEventTopics(getEventTopicList(apiList))
+            eventTypesDao.insertEventTypes(getEventTypeList(apiList))
```

Comment:
@iamareebjamal i'm saving individually as well so there is a table for them as well

Reply:
Why are you converting in JSON then? Just save them as JSON. No need to add tables

Reply:
okay i'll remove them

Reply:
updated